app/public/js/system/userConfig.js

```diff
@@ -49,4 +49,38 @@ userConfig.saveWindow = function(width, height) {
         window.localStorage.width = Math.round(width);
         window.localStorage.height = Math.round(height);
     }
-};
\ No newline at end of file
+};
+
+userConfig.scaleWindow = function(scale){
```

Comment:
Can we in a way remove the conditions here because you can perhaps just send the value number from the shortcuts method e.g `scaleWindow(-0.1)` or `scaleWindow(0.1)`